unity: android crash handler (#11757)
getsentry/sentry-docs: docs/platforms/unity/native-support/index.mdx

When choosing Android as a build target, Unity creates a Gradle project. During the Unity build process, the SDK embeds the [Android SDK](/platforms/android/), as well as the provided options, by adding it to the generated project's Android Manifest.

> Note that [Unity has an Android crash handler which takes treats the crash information as a string](https://docs.unity3d.com/6000.0/Documentation/Manual/android-handle-crashes.html). This doesn't allow Sentry to symbolicate the crash and in result becomes a very low quality crash dump.
> We recomment disalbing that through: `-androidChainedSignalHandlerBehavior:disabled`. For more information, [read the Unity documentation on this topic](https://docs.unity3d.com/6000.0/Documentation/Manual/android-handle-crashes.html).
> 

Reviewer: ```suggestion
Note, that Unity has an [Android crash handler](https://docs.unity3d.com/6000.0/Documentation/Manual/android-handle-crashes.html), which treats crash information as a string. This doesn't allow Sentry to symbolicate the crash, which becomes a very low-quality crash dump as a result. We recommend disabling it through: `-androidChainedSignalHandlerBehavior:disabled`. For more information, [read the Unity documentation](https://docs.unity3d.com/6000.0/Documentation/Manual/android-handle-crashes.html).

```
